lib/mdl/rules.rb

```diff
@@ -190,11 +190,11 @@
       |(l, e), i| (i + 1 < locations.size ?
                    (l..locations[i+1].first - 1) :
                    (l..doc.lines.count)).to_a if e.type == :table }.flatten
-    if params[:hard_wrap]
+    if params[:style] == :wrapped
       violation_lines = doc.matching_lines(/^.{#{@params[:line_length]}}.*\s/)
       violation_lines -= codeblock_lines unless params[:code_blocks]
       violation_lines -= table_lines unless params[:tables]
-    else
+    elsif params[:style] == :not_wrapped
```

Comment:
can you add an `else` and then `raise` an exception in case someone passes somethign else in?

Reply:
Sure, I wasn’t sure what to do about that—I don’t like quiet failures. I’ll get this change made.